Automattic/wp-calypso
# Help Center: Add Clear Conversation option to Help Center chat header

Fixes: https://github.com/Automattic/dotcom-forge/issues/9545

## Proposed Changes

TODO: The chat is cleared, but I need to figure out how to fix the page refresh to see the new chat immediately.

* Adds ellipsis menu to header with `Clear chat` option when the user is in a chat.
* Adds `clear chat` option to clear and start a new support chat.

## Why are these changes being made?


* Allow users to start a new chat.

## Testing Instructions



* Checkout branch or use live link.
* Navigate to `/home?flags=help-center-experiences`
* Open the Help Center, open a chat and verify the ellipsis menu appears.
* The clear chat button and verify it clears/starts a new chat.

## Pre-merge Checklist
